src/components/WalletProviderModal/WalletProviderModal.module.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../styles/variables";
+
+.container {
+  .provider {
+    cursor: pointer;
+    padding: $spacing-xl;
+    border-bottom: 1px solid white;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(136.33deg, rgba(135, 22, 232, 0.28) 7.74%, rgba(254, 110, 153, 0) 98.49%);
+
+    box-sizing: border-box;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
I've created `Column` and `Row` components to avoid repeating flexbox css rules. Maybe we can used them here instead? Checkout `Fundraising.tsx` as an example.

Reply:
Damn, I totally forgot about them. I will refactor them in a bit :D